src/lib/booleans.js

```diff
@@ -0,0 +1,62 @@
+exports.negate = a => {
+  return !a;
+};
+
+exports.both = (a, b) => {
+  return a && b;
+};
+
+exports.either = (a, b) => {
+  return a || b;
+};
+
+exports.none = (a, b) => {
+  if (!a && !b) {
+    return true;
+  }
+  return false;
+};
+
+exports.one = (a, b) => {
+  return a != b;
+};
+
+exports.truthiness = a => {
+  return !!a;
+};
+
+exports.isEqual = (a, b) => {
+  return a == b;
+};
+
+exports.isGreaterThan = (a, b) => {
+  return a > b;
+};
+
+exports.isLessThanOrEqualTo = (a, b) => {
+  return a <= b;
+};
+
+exports.isOdd = a => {
+  return a % 2 !== 0;
+};
+
+exports.isEven = a => {
+  return a % 2 == 0;
```

Comment:
sneaky == again, just make sure you want this to be coerced, because if not always use ===. I have never used == in my professional career so far, so tend to steer clear as === is more precise.